lib/browse_everything/driver/kaltura.rb

```diff
@@ -52,6 +59,15 @@ def authorized?
         true
       end
 
+      def structure_transform
+         val = @@entries.to_h
+         @@entries = Hashie::Mash.new(val)
+         @@entries.extend(::Kaltura::Extensions::Mash)
+         convert_array = Array.new
+         convert_array[0] = @@entries
+         @@entries = convert_array
```

Comment:
Just a style thing:  

```
convert_array = Array.new
convert_array[0] = @@entries
@@entries = convert_array
```
can be accomplished with just `@@entries = [@@entries]`

Reply:
Thanks.  This works!